From 4fa081a3972c4975e74f5a194733a585a6b76db4 Mon Sep 17 00:00:00 2001
From: Kurt Raschke <kurt@kurtraschke.com>
Date: Sun, 12 Oct 2025 21:48:15 -0400
Subject: [PATCH] Begin testing on Python 3.14.

Support for Python 3.9 is expected to be dropped at the end of the month.
---
 pyproject.toml      | 6 ++++--
 pyrfc3339/parser.py | 2 +-
 2 files changed, 5 insertions(+), 3 deletions(-)

diff --git a/pyrfc3339/parser.py b/pyrfc3339/parser.py
index e9eceaa..7bd519d 100644
--- a/pyrfc3339/parser.py
+++ b/pyrfc3339/parser.py
@@ -36,7 +36,7 @@ def parse(timestamp: str, utc: bool = False, produce_naive: bool = False) -> dat
 
     Exceptions will, however, be thrown for blatantly invalid input:
 
-    >>> parse('2009-01-01T25:01:02Z')
+    >>> parse('2009-01-01T25:01:02Z') # doctest: +IGNORE_EXCEPTION_DETAIL
     Traceback (most recent call last):
     ...
     ValueError: hour must be in 0..23
